.eslintrc

```diff
@@ -1,12 +1,12 @@
 {
-  "extends": ["airbnb", "plugin:inferno/recommended"],
```

Comment:
I dropped airbnb because of the dependencies with react. But I agree, we should have a strict lint. Have started working with base and will update this PR :)